Classes/Controller/ApiController.php

```diff
@@ -71,6 +71,12 @@ class ApiController extends ActionController
      */
     protected $prototypeDetailsFactory;
 
+    /**
+     * @see http://www.iana.org/assignments/media-types/index.html
+     * @var array
+     */
+    protected $supportedMediaTypes = ['application/json'];
```

Comment:
Is this really necessary, it will lead to problems when the url is called in a browser for debuging proposes since the browser will request html. 

Answering with application/json to an html request is fine.

Reply:
Good point. I removed it.